trace/Sampling.md

```diff
@@ -8,7 +8,31 @@ exporters.
 
 The Sampling bit is always set only at the start of a Span, using a `Sampler`
 
-### What kind of samplers does OpenCensus support?
+### Sampler interface
+A `Sampler` is an interface with only one method `sample` that accepts a set of sampling
+parameters and returns a `SamplingDecision` object.
+
+#### Sampling parameters
+Based on the language the sampling parameters can be embedded into an object called
+`SamplingParameters` or explicitly passed as arguments. The list of sampling parameters SHOULD
+include at least:
+* The parent `SpanContext`;
+* The current `TraceId`;
+* The current `SpanId`;
+* The current `Span`'s name;
+* The current `Span`'s kind;
```

Comment:
Some implementations also have HasRemoteParent, e.g. https://godoc.org/go.opencensus.io/trace#SamplingParameters. I don't have the historical background what it is but FYI.

Reply:
After we added the Span.kind that argument is not that useful. We need to decide if we still need it. As the current text mention the least can include more sampling params if needed.